Engine/GraphicsCore.cs

```diff
@@ -355,11 +359,18 @@ private static void GeometryPass(Camera camera)
                                                                                                                    camera.GBuffer.normal.GetView<RenderTargetView>(),
                                                                                                                    camera.GBuffer.metallic.GetView<RenderTargetView>(),
                                                                                                                    camera.GBuffer.roughness.GetView<RenderTargetView>(),
-                                                                                                                   camera.GBuffer.ambientOcclusion.GetView<RenderTargetView>());
+                                                                                                                   camera.GBuffer.ambientOcclusion.GetView<RenderTargetView>(),
+                                                                                                                   camera.GBuffer.emission.GetView<RenderTargetView>());
             
             CurrentDevice.ImmediateContext.ClearRenderTargetView(camera.GBuffer.worldPos.GetView<RenderTargetView>(), new RawColor4(0.0f, 0.0f, 0.0f, 0.0f));
+            CurrentDevice.ImmediateContext.ClearRenderTargetView(camera.GBuffer.albedo.GetView<RenderTargetView>(), new RawColor4(0.0f, 0.0f, 0.0f, 0.0f));
```

Comment:
there is no need to clear every buffer every frame as worldPos and depth buffer is enough to determine if there is an object on a pixel